src/server/project.js

```diff
@@ -0,0 +1,291 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+
+var fs = require('fs'),
+    path = require('path'),
+    cordovaServe = require('cordova-serve'),
+    Q = require('q'),
+    dirs = require('./dirs'),
+    pluginUtil = require('./utils/plugins'),
+    prepareUtil = require('./utils/prepare'),
+    utils = require('./utils/jsUtils');
+
+/**
+ * @constructor
+ */
+function Project(simulator, platform, debugHostHandlers) {
+    this._simulator = simulator;
+    this.platform = platform;
+    this.debugHostHandlers = debugHostHandlers;
```

Comment:
`debugHostHandlers` was duplicated, so I removed from the `Project` model in favor of keeping it as part of the `Simulator` model, since it looks like more to a configuration related to the simulation itself rather than the project.
Thanks!